pkg/scheduler/api/well_known_labels.go

```diff
@@ -57,15 +57,6 @@ const (
 	// and removed when node has enough disk.
 	TaintNodePIDPressure = "node.kubernetes.io/pid-pressure"
 
-	// TaintExternalCloudProvider sets this taint on a node to mark it as unusable,
-	// when kubelet is started with the "external" cloud provider, until a controller
-	// from the cloud-controller-manager intitializes this node, and then removes
-	// the taint
-	TaintExternalCloudProvider = "node.cloudprovider.kubernetes.io/uninitialized"
```

Comment:
I can't recall the issue, but wasn't there some ongoing work to move schedulerapi somewhere else that's more easily consumable? Maybe @ahg-g knows?

Reply:
https://github.com/kubernetes/kubernetes/issues/82390 is the issue. these taints are only used by the cloud-provider control loops, so it seems natural to move these here

Reply:
What I like about this location is that the taints are considered APIs, in the new location it seems easier for someone to change them without understanding the implications. Wondering if we should have a `k8s.io/cloud-provider/api` package or something? We have other constants that might be a better fit there -- see [here](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cloud-provider/service/helpers/helper.go#L33-L38) and [here](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cloud-provider/volume/constants.go#L19-L26). Thoughts @cheftako?

Reply:
I'm fine with `/api`

Reply:
here is the api cleanup issue: https://github.com/kubernetes/kubernetes/pull/83578, well_known_labels.go seems to be the only file that will stay under scheduler/api, @Huang-Wei what is the plan for this file?

Reply:
> Wondering if we should have a `k8s.io/cloud-provider/api` package or something? We have other constants that might be a better fit there -- see [here](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cloud-provider/service/helpers/helper.go#L33-L38) and [here](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cloud-provider/volume/constants.go#L19-L26). Thoughts @cheftako?

seems reasonable to me

Reply:
@ahg-g Here is a list which refers `k8s.io/kubernetes/pkg/scheduler/api`:

```
plugin/pkg/admission/defaulttolerationseconds/admission.go
28:	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"

plugin/pkg/admission/podtolerationrestriction/admission.go
38:	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"

pkg/scheduler/algorithm/predicates/predicates.go
46:	legacyapi "k8s.io/kubernetes/pkg/scheduler/api"

pkg/scheduler/algorithm/types.go
26:	legacyapi "k8s.io/kubernetes/pkg/scheduler/api"

pkg/scheduler/api/BUILD
11:    importpath = "k8s.io/kubernetes/pkg/scheduler/api",

pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go
45:	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"

pkg/controller/daemon/util/daemonset_util.go
29:	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"

pkg/controller/cloud/node_lifecycle_controller.go
40:	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"

pkg/controller/cloud/node_controller.go
41:	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"

pkg/controller/nodelifecycle/node_lifecycle_controller.go
63:	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"

pkg/kubelet/kubelet_node_status.go
44:	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"

pkg/kubelet/eviction/eviction_manager.go
43:	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"
```

It seems the usage are not quite related with cloud-provider, so I'd suggest to move the const strings in `pkg/scheduler/api/well_known_labels.go` to a more common place, such as `staging/src/k8s.io/api/core/v1/well_known_labels.go`. WDYT?